docs/source/tracking.rst

```diff
@@ -370,6 +370,36 @@ This path must the same on both the server and the client -- you may need to use
 the client in order to enforce this property.
 
 
+HDFS
+^^^^^
+
+To store artifacts in an HDFS, specify a URI of the form ``hdfs:/<path>``. It can contain host and port:
+
+|     ``hdfs:/localhost:8020/path`` or just the path:
```

Comment:
Technically this is supposed to be `hdfs://localhost:2020/path` I'm pretty sure. If you only use one `/` then the host:port is treated as a path name.